.gitlab-ci.yml

```diff
@@ -569,6 +569,11 @@ run_dogstatsd_arm_size_test:
 
 .system-probe_build_common:
   before_script:
+    # HACK: install gimme so we can build with go 1.10.1
```

Comment:
PR to add this to the builders so we can remove this hack:
https://github.com/DataDog/datadog-agent-buildimages/pull/32